### ISSUE TITLE
Unlimited upvotes

### ISSUE THREAD

**Issue by ThreeFx** [timakin/codemagnet_app#9]
I do not know if this is intended but please restrict users to only upvote once. Otherwise the best posts will not necessarily have the highest score.

**Comment by timakin** [comment]
@ThreeFx Hi, I tried this setting just for fun, but it looks not good, so we'll change asap!! The highest priority.

**Comment by shogochiai** [comment]
How do you plan to solve this problem? Will upvoter be required login?

**Comment by timakin** [comment]
@shogochiai yep. we gonna add github singin system and try to limit 1 time upvoting for logged in user.

**Comment by ThreeFx** [comment]
A login is overkill IMO. Multiple upvpting would only have to be suffieciently complex to "scare" the people off.

**Comment by timakin** [comment]
@ThreeFx So, we should deal with cookie data to limit upvoting :)
I also think logging is problem.

**Comment by snghrsw** [comment]
completed

**Comment by shogochiai** [comment]
@snghrym LGTM, Nice localStorage idea!
